clients/src/main/resources/common/message/ListOffsetsRequest.json

```diff
@@ -36,7 +36,9 @@
   // Version 8 enables listing offsets by local log start offset (KIP-405).
   //
   // Version 9 enables listing offsets by last tiered offset (KIP-1005).
-  "validVersions": "0-9",
+  //
+  // Version 10 enables async remote list offsets support (KIP-1075)
+  "validVersions": "0-10",
```

Comment:
since we have flexible versions from v6, is it required to bump the version to 10?